exchange/howto/list-asset.adoc

```diff
@@ -44,13 +44,13 @@ https://github.com/bisq-network/bisq/issues/new?template=new_asset.md[Create iss
 
 **You must use the link above to create the issue, as it links to a template designed specifically for new asset requests.**
 
-WARNING: Creating an issue to list a new asset is not a guarantee that your asset will be listed. It's a request to convince a Bisq developer to spend the time needed to implement the new asset, write tests, etc. You can help by making a request that is complete and well-formed, and by being cooperative throughout the process.
+WARNING: Creating an issue to list a new asset is not a guarantee that your asset will be listed. It's a request to convince a Bisq developer to spend the time needed to implement the new asset, write tests, etc. You can help by making a request that is complete and well-formed, and by being cooperative throughout the process. Bisq is always short with devs. Offering a bounty is a good idea to help your asset being listed.
```

Comment:
```suggestion
WARNING: Creating an issue to list a new asset is not a guarantee that your asset will be listed. It's a request to convince a Bisq developer to spend the time needed to implement the new asset, write tests, etc. You can help by making a request that is complete and well-formed, and by being cooperative throughout the process. Bisq developers have their hands full, and adding assets is not usually a top priority, so offering a bounty can help your asset get listed.
```

Reply:
I find that a bit long, but for me the essential point is to have that strongly suggested. So it's ok for me.
I have another (old) idea to add, but we could discuss that maybe on the slack.

Reply:
It's longer but I think the English is a bit better.